### ISSUE TITLE
[FEATURE] Daemonize teler

### ISSUE THREAD

**Issue by dwisiswant0** [kitabisa/teler#276]
### Discussed in https://github.com/kitabisa/teler/discussions/274

<div type='discussions-op-text'>

<sup>Originally posted by **0xjmux** December  4, 2022</sup>
I'm trying to run teler against an nginx access log, like the below from the docs:
```
$ tail -f /var/log/nginx/access.log | teler -c /path/to/config/teler.yaml
```
However, all the resources I've found on the wiki/site have you run it in an active terminal session, or only analyze logs at a specific point in time. I'd like teler to analyze logs in real time, as they come in, and export it's own log in real time so it can be picked up by my SIEM system - which is configured to monitor teler's log file `/var/log/teler.log`. 

My many attempts to convert `teler` into a systemd service have failed - you can't have pipes in service commands, for one. I won't go through all the things I've tried, but I figure there has to be a way to run teler as a service that can be set up to run in the background, instead of live in the foreground taking up a tty. 

Is there a way to do this, and if so, how? Thanks. </div>

**Comment by dwisiswant0** [comment]
Dependent on #277.

**Comment by clavinjune** [comment]
how about changing the input from `stdin` to `-f files-name`? We can do tailing inside teler instead of depends on `tail -f`

for example:

```shell
$ teler -c /path/to/config/teler.yaml -f /var/log/nginx/access.log
```

[reference](https://github.com/hpcloud/tail)

**Comment by dwisiswant0** [comment]
> how about changing the input from `stdin` to `-f files-name`? We can do tailing inside teler instead of depends on `tail -f`

Please see https://github.com/kitabisa/teler/issues/277.

**Comment by asheroto** [comment]
@dwisiswant0 have you guys considered hosting the docs locally? I'd love to contribute. Or maybe there's docs repo I'm missing?

Here's a simple example of daemonizing with automatic reloading on config change:

https://gist.github.com/asheroto/ff192448bb27b27e7c2d7cefdc24f308

**Comment by dwisiswant0** [comment]
@asheroto - Hi, "the development of teler IDS is currently on hold"[^1] - this announcement was made last week through `teler-announce@googlegroups`. Your input regarding local development documentation has been duly noted, and we will be addressing it by migrating _(or completely rewriting)_ the docs.

[^1]: [[teler] Version 3 Roadmap 📝 [survey]](https://groups.google.com/g/teler-announce/c/Fm8Yjh_4of0)

**Comment by dwisiswant0** [comment]
I'm closing this issue as not planned.
